pkg/sql/sem/builtins/builtins_test.go

```diff
@@ -125,13 +126,20 @@ func TestGenerateUniqueUnorderedIDOrder(t *testing.T) {
 
 	// We test our null hypothesis by running 100 trials and then performing
 	// a z-test (https://en.wikipedia.org/wiki/Z-test) for the observed mean
-	// number of swaps. We then compare it against the critical value of +-1.96,
-	// which corresponds to a confidence level of 95%.
+	// number of swaps.
 	const (
 		numTrials       = 100
 		numGensPerTrial = 101
 	)
 
+	// We then compare it against the critical value of +-2.58,
+	// which corresponds to a confidence level of 99%.
```

Comment:
would a confidence level of 99% make this more likely to flake?

Reply:
It would be less likely to flake since a higher confidence level means we are even more certain that the mean is different (i.e. there's a bigger margin of error allowed)